releasenotes/notes/Optimize1qGates-basis-aware-34cede46472c3826.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    The pass ``Optimize1qGates`` now supports an optional parameter ``basis_gates``.
+    This allows ``Optimize1qGates`` to avoid using gates that are no supported by the basis.
```

Comment:
i'm not sure this feature is worth highlighting in a release note, i see it as mostly a hack to fix the bug. it's not a scalable solution since it only works for (some) subsets of the {'u1','u2','u3'} set

Reply:
release note removed in 852e8acd38d16f49f8954d3880cc7af560188d5c